### PR TITLE
Update rubygem-foreman_ansible to 7.0.1

### PR DESCRIPTION
**Pull request by ezr-ondrej**
Should get into 3.1

RPM: #7387

### CODE REVIEW

**Comment by evgeni** [comment]
This is now failing on the graphql loader issue.

You can workaround it like we did in https://github.com/theforeman/foreman-packaging/pull/7375 until we properly fix https://projects.theforeman.org/issues/34086

**Comment by ezr-ondrej** [comment]
We include the `mk` there :(

**Comment by ezr-ondrej** [comment]
:hankey: :sweat_drops:

**Comment by evgeni** [comment]
Oohhh, right, I used `foreman_ansible` as my playground for the shared Makefile… Let me circle back to Avi for the proper fix in core then.

File: debian/buster/foreman/foreman-plugin.mk [hunk 1]
```diff
@@ -4,6 +4,7 @@ build:
 	cd /usr/share/foreman && ( \
 		/usr/bin/foreman-ruby /usr/bin/bundle install --local && \
 		if [ "$(PLUGIN_HAS_PACKAGE_JSON)" = "true" ]; then /usr/bin/npm install --no-audit --no-optional --unsafe-perm; fi && \
+		if [ "$(PLUGIN_NEEDS_GRAPHQL_WORKAROUND)" = "true" ]; then /usr/bin/npm install --save-dev 'graphql@^15.5.0' 'graphql-tag@^2.11.0'; fi && \
```
Comment (evgeni) [inline debian/buster/foreman/foreman-plugin.mk:7]: I don't think we should codify this workaround so nicely, but rather fix the actual issue via https://github.com/theforeman/foreman/pull/8975

**Comment by evgeni** [comment]
Needs https://ci.theforeman.org/job/foreman-develop-source-release/1458/ to finish and then new packages built from that.

**Comment by evgeni** [comment]
Which happened now, so:

[test deb]

**Comment by ezr-ondrej** [comment]
:green_apple: :tada:

**Review by evgeni — APPROVED** [review]

**Comment by evgeni** [comment]
Thanks for flying "Evgeni yells at NPM" Air!

**Comment by ezr-ondrej** [comment]
CP 3.1: #7399

**Pull request merged**